isce2_topsapp/localize_slc.py

```diff
@@ -83,9 +80,10 @@ def get_interferogram_geo(
     # Update the area of interest based on frame_id
     if frame_id != -1:
         gunw_geo = get_gunw_extent_from_frame_id(frame_id)
-        if not gunw_geo.intersects(ifg_geo):
+        frame_coverage = gunw_geo.intersection(ifg_geo).area / gunw_geo.area
+        if frame_coverage < min_frame_coverage:
             raise ValueError(
```

Comment:
It would be helpful to raise an error indicating whether the reference or secondary date fails to meet the spatial coverage requirement, to prevent submitting a date with missing coverage or gaps (e.g., due to a MultiPolygon).